frontend/public/components/overview/project-overview.tsx

```diff
@@ -13,6 +13,11 @@ import {
   PodControllerOverviewItem,
   OverviewItem,
 } from '@console/shared';
+import {
+  AlertSeverityIcon,
+  getSeverityAlertType,
+  getFiringAlerts,
+} from '@console/dev-console/src/components/topology';
```

Comment:
i don't we should import from `dev-console` in public cc @jeff-phillips-18

Reply:
moved the utils to `console-shared/src/utils/alert-utils.ts `